msteams-platform/m365-apps/actions-in-m365-quick-start-guide.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Actions in Microsoft 365 quick start guide
+description: In this article, learn how to create Actions in Microsoft 365, function of Actions and its use cases.
+ms.date: 12/8/2023
+ms.author: mosdevdocs
+author: v-preethah
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365 quick start guide
+
+Get started with Actions in Microsoft 365 using the Actions across Microsoft 365 sample.
+
+## Prerequisites
+
+Before you get started, ensure that you install the following tools:
+
+| &nbsp; | Install | For using... |
```

Comment:
Same as earlier comment. I dont think the description in column "For using" are accurate. If there aren't doc pattern we must follow, I feel we don't need write "for using" for each of it.